pages/blog/posts/bowtie-intro.md

```diff
@@ -146,7 +146,7 @@ This is probably the most beneficial area you can help in.
 I myself am not a frontend developer, in case that wasn't obvious.
 What I put together (with Bootstrap) is essentially the bare minimum of what's needed to show off results Bowtie emits.
 If you have more experience or headspace to think about how to effectively report test results, or compare implementations, please help!
-All of Bowtie's "frontend code" today lives in one place, [a Jinja2 template which gets formatted into a static single page site](https://github.com/bowtie-json-schema/bowtie/blob/main/bowtie/templates/report.html.j2).
```

Comment:
This isn't true anymore, but was true at the time of the post. Link-wise/stylistically I think the proper thing to do is probably just to link to a commit from the time, so that'd be e.g. https://github.com/bowtie-json-schema/bowtie/blob/b23644eb86cf9bcbdf1d98149cf3d976339f7cdf/bowtie/templates/report.html.j2

This sort of thing is why some blogs do "post updated on XYZ" things, so if we wanted to correct the *information* that seems like a decent thing to do but possibly relies on some infrastructure we may not have yet on the frontend (i.e. the ability to have multiple versions of the post, or at least to add that "updated on" note). I'd probably just change the link here for now and if we really cared we can come back later to make the note about accuracy because there undoubtedly are a bunch of things in this post which are now fixed/no longer true.

Reply:
@Julian For now I have just changed the link. I do not have deep understanding of the current infrastructure but we could just add another field to the markdown gray matter titled `last-updated` and just make desired changes, set the updated date and add this new updated date component somewhere next to the first release date.